crates/turbo-tasks/src/id.rs

```diff
@@ -77,6 +77,14 @@ impl Debug for TaskId {
     }
 }
 
+pub const TRANSIENT_TASK_BIT: u32 = 0x8000_0000;
```

Comment:
We could go for `0xc000_0000` if we need more space for persistent tasks. That would work with the system, but I keep it in the middle for now